bitnami/minio/values.yaml

```diff
@@ -135,6 +135,10 @@ tls:
   enabled: false
   ## Name of an existing secret holding the certificate information
   ##
+  existingSecret:
+  ## DEPRECATED. Use `tls.existingSecret` instead.
```

Comment:
I think this is confusing, could you explain a little more?

Reply:
nice!

Reply:
Hi @Mauraza! 

In addition to adding the autogenerated certificates feature to the charts, I'm also trying to standardize the settings when backward compatibility is possible.

In this case, I'm deprecating `.Values.tls.secretName` and adding the new standard value `.Values.tls.existingSecret` included in other charts.

I considered `existingSecret` would fit better because it is already being used for authentication settings in most of the charts (`auth.existingSecret`), and it would be appropriate considering we now provide two options:
- Use an 'existingSecret' containing the certificates manually generated.
- Autogenerate the certificates secret if `tls.autoGenerated=true`.

Otherwise, a variable name such 'secretName' could be confused with an input variable used, for example, to specify the name of the autogenerated secret.